recipes/pytest-timeout/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: pytest-timeout
+  version: '{{ version }}'
+
+source:
+  fn: '{{ version }}.tar.gz'
+  url: https://bitbucket.org/pytest-dev/pytest-timeout/get/{{ version }}.tar.gz
+  sha256: 0c69eb6d038ca02445bd83921da148bd274b25c5e9cebcad58860f9f05de9e6f
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest >=2.8
```

Comment:
I think you need `setuptools` here too (or maybe only for the tests so them can find `pkg_resources`.)